README.md

```diff
@@ -1,4 +1,4 @@
-# React Navigation website
+<h2 align="center"><a href="https://reactnavigation.org/"> <img src="https://github.com/vivekverma007/react-navigation.github.io/blob/source/static/img/react_navigation_header.png" width="400" /> </a></h2>
```

Comment:
Hi! You are referencing to a direct URL to your own fork, please use a relative URL instead

Reply:
### I have Updated the Readme.md with relative Image Path